File: Java/src/main/java/knowledge_layer/SystemModelManager.java
package knowledge_layer;

import specification.SystemModel;

public class SystemModelManager {
    public SystemModel sysModel;

    public SystemModelManager(SystemModel sys) {
        sysModel = sys;
    }

    public void updateModel() {
        //todo: specify how to update model
    }

    public SystemModel getSysModel() {
        return sysModel;
    }
}

File: Python/knowledge_layer/SystemModelManager.py
from specification.SystemModel import SystemModel


class SystemModelManager:

    def __init__(self, sys: SystemModel):
        self.sysModel = sys

    def updateModel(self):
        # TODO: Specify how to update model
        pass

    def getSysModel(self) -> SystemModel:
        return self.sysModel
